src/components/editor/property/NestedPropertyHeader.jsx

```diff
@@ -97,6 +98,7 @@ NestedPropertyHeader.propTypes = {
   hideProperty: PropTypes.func,
   id: PropTypes.string,
   resourceKey: PropTypes.string,
+  propertyLabelId: PropTypes.string,
```

Comment:
i couldn't tell from `grep`ing the code how this would _not_ be provided, but when i marked it required, i got test failures.  happy to revisit if there was something in the tests i should've fixed.